plugins/Microsoft.DocAsCode.Build.UniversalReference/BuildOutputs/ApiBuildOutput.cs

```diff
@@ -0,0 +1,142 @@
+
+    using Microsoft.DocAsCode.DataContracts.Common;
+    using Microsoft.DocAsCode.YamlSerialization;
+
+    [Serializable]
+    public class ApiBuildOutput
+    {
+        [YamlMember(Alias = Constants.PropertyName.Uid)]
+        [JsonProperty(Constants.PropertyName.Uid)]
+        public string Uid { get; set; }
+
+        [YamlMember(Alias = Constants.PropertyName.CommentId)]
+        [JsonProperty(Constants.PropertyName.CommentId)]
+        public string CommentId { get; set; }
+
+        [YamlMember(Alias = Constants.PropertyName.Parent)]
+        [JsonProperty(Constants.PropertyName.Parent)]
+        public ApiListInDevlangs<ApiNames> Parent { get; set; }
+
+        [YamlMember(Alias = Constants.PropertyName.Children)]
+        [JsonProperty(Constants.PropertyName.Children)]
+        public ApiListInDevlangs<List<ApiBuildOutput>> Children { get; set; }
+
+        [YamlMember(Alias = Constants.PropertyName.Href)]
+        [JsonProperty(Constants.PropertyName.Href)]
+        public string Href { get; set; }
+
+        [YamlMember(Alias = "langs")]
+        [JsonProperty("langs")]
+        public string[] SupportedLanguages { get; set; }
+
+        [YamlMember(Alias = Constants.PropertyName.Name)]
+        [JsonProperty(Constants.PropertyName.Name)]
+        public ApiListInDevlangs<string> Name { get; set; }
+
+        [YamlMember(Alias = Constants.PropertyName.NameWithType)]
+        [JsonProperty(Constants.PropertyName.NameWithType)]
+        public ApiListInDevlangs<string> NameWithType { get; set; }
+
+        [YamlMember(Alias = Constants.PropertyName.FullName)]
+        [JsonProperty(Constants.PropertyName.FullName)]
+        public ApiListInDevlangs<string> FullName { get; set; }
+
+        [YamlMember(Alias = Constants.PropertyName.Type)]
+        [JsonProperty(Constants.PropertyName.Type)]
+        public string Type { get; set; }
+
+        [YamlMember(Alias = Constants.PropertyName.Source)]
+        [JsonProperty(Constants.PropertyName.Source)]
+        public ApiListInDevlangs<SourceDetail> Source { get; set; }
+
+        [YamlMember(Alias = Constants.PropertyName.Documentation)]
+        [JsonProperty(Constants.PropertyName.Documentation)]
+        public SourceDetail Documentation { get; set; }
+
+        [YamlMember(Alias = Constants.PropertyName.Assemblies)]
+        [JsonProperty(Constants.PropertyName.Assemblies)]
+        public ApiListInDevlangs<List<string>> AssemblyNameList { get; set; }
+
+        [YamlMember(Alias = Constants.PropertyName.Namespace)]
+        [JsonProperty(Constants.PropertyName.Namespace)]
+        public ApiListInDevlangs<ApiNames> NamespaceName { get; set; }
+
+        [YamlMember(Alias = "summary")]
+        [JsonProperty("summary")]
+        public string Summary { get; set; } = null;
+
+        [YamlMember(Alias = "remarks")]
+        [JsonProperty("remarks")]
+        public string Remarks { get; set; }
+
+        [YamlMember(Alias = "example")]
+        [JsonProperty("example")]
```

Comment:
want to keep the same name with the one in mref

---
In reply to: [113360636](https://github.com/dotnet/docfx/pull/1596#discussion_r113360636) [](ancestors = 113360636)